src/routes/users/posts.ts

```diff
@@ -0,0 +1,11 @@
+import express, { Router } from 'express';
```

Comment:
create the `posts.ts` file as a direct child to the `routes` directory (`src/routes/posts.ts`)